plugins/autocomplete/plugin.js

```diff
@@ -148,20 +148,15 @@
 	 *	} );
 	 * ```
 	 * @param {CKEDITOR.editor} editor The editor to watch.
-	 * @param {Function} textTestCallback Callback executed to check if a text next to the selection should open
-	 * the autocomplete. See the {@link CKEDITOR.plugins.textWatcher}'s `callback` argument.
-	 * @param {Function} dataCallback Callback executed to get suggestion data based on search query. The returned data will be
-	 * displayed in the autocomplete view.
-	 * @param {String} dataCallback.query The query string that was accepted by the `textTestCallback`.
-	 * @param {CKEDITOR.dom.range} dataCallback.range The range in the DOM where the query text is.
-	 * @param {Function} dataCallback.callback The callback which should be executed with the data.
-	 * @param {CKEDITOR.plugins.autocomplete.model.item[]} dataCallback.callback.data The suggestion data that should be
-	 * displayed in the autocomplete view for a given query. The data items should implement the
-	 * {@link CKEDITOR.plugins.autocomplete.model.item} interface.
-	 * @param {String} [itemTemplate] Template for list item in dropdown. See {@link CKEDITOR.plugins.autocomplete.view#itemTemplate} for more information.
-	 * @param {String} [outputTemplate] Template for match rendering. See {@link #outputTemplate}.
+	 * @param {CKEDITOR.plugins.autocomplete.configDefinition} config Configuration object keeping information
+	 * how to instantiate autocomplete plugin.
```

Comment:
> instantiate autocomplete plugin

It's not instantiating the plugin, it's more of a component. I'd just simpify it to "Configuration object for this autocomplete instance." or something alike.